src/main.c

```diff
@@ -49,7 +49,7 @@ main (int argc, char *argv[])
   };
 
   short int next_option = 0;
-
+ 
```

Comment:
[Trailing white space](http://programmers.stackexchange.com/questions/121555/why-is-trailing-whitespace-a-big-deal) needs to be removed. In Geany, check the box view -> show whitespace